proc/threads.c

```diff
@@ -841,6 +876,97 @@ int proc_threadCreate(process_t *process, void (*start)(void *), unsigned int *i
 }
 
 
+static unsigned int _proc_lockGetPriority(lock_t *lock);
```

Comment:
No need for this, just move funtion body here

Reply:
Done.